site/src/kiosk/kiosk-use-case.js

```diff
@@ -15,15 +16,15 @@ export const watchAnonymousStateChange = (dispatch, auth) => () => {
 
 export const watchDeviceContent = (dispatch) => (deviceId) => {
     return watchDeviceData(deviceId, (url) => {
-        fetch(url)
-            .then(response => response.text())
-            .then(html => {
-                dispatch(Actions.onDeviceContent(html))
-            })
+        loadHtml(url).then(html => {
+            dispatch(Actions.onDeviceContent(html))
+        }).catch((error) => {
```

Comment:
just a nitpick, but i've seen some declarations where you wrap the single param with `()` and others where you don't. maybe we could setup a linter? 😄

Reply:
I'm terrible for this, yeah I agree. Do you have a preference for which one to choose? Always wrap or only for multiple params?

Reply:
my personal preference is only for multiple params. but don't mind if you go the other way.